src/status_im/ui/screens/events.cljs

```diff
@@ -263,6 +263,7 @@
                                               [:load-accounts]
                                               [:listen-to-network-status]
                                               [:navigate-to :accounts]]}
+                                (navigation/navigate-to-clean nil)
```

Comment:
What will navigating to `nil` achieve ?

Reply:
@janherich we go back to the initial state and removes the subscription (database starts with `nil` in `view-id`). 

Basically if we don't subscriptions as still fired on database changes, but `account/account` (for example) is not there. 

In this example after `re-frame` process `:initialize-db` it will try to render the current `:view-id` but has not the correct data. Does it make sense?